apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/test/java/gov/cms/bfd/pipeline/rda/grpc/server/RandomMcsClaimGeneratorTest.java

```diff
@@ -53,88 +53,265 @@ public void randomClaim() throws InvalidProtocolBufferException {
             + "  \"mcsDiagnosisCodes\": [{\n"
             + "    \"idrClmHdIcn\": \"84555\",\n"
             + "    \"idrDiagIcdTypeEnum\": \"DIAGNOSIS_ICD_TYPE_ICD9\",\n"
-            + "    \"idrDiagCode\": \"b9b6\"\n"
-            + "  }, {\n"
-            + "    \"idrClmHdIcn\": \"84555\",\n"
-            + "    \"idrDiagIcdTypeEnum\": \"DIAGNOSIS_ICD_TYPE_ICD10\",\n"
-            + "    \"idrDiagCode\": \"v3\"\n"
-            + "  }, {\n"
-            + "    \"idrClmHdIcn\": \"84555\",\n"
-            + "    \"idrDiagCode\": \"f24\",\n"
-            + "    \"idrDiagIcdTypeEnumUnrecognized\": \"k\"\n"
-            + "  }, {\n"
-            + "    \"idrClmHdIcn\": \"84555\",\n"
-            + "    \"idrDiagIcdTypeEnum\": \"DIAGNOSIS_ICD_TYPE_ICD9\",\n"
-            + "    \"idrDiagCode\": \"xp3k0b\"\n"
-            + "  }, {\n"
-            + "    \"idrClmHdIcn\": \"84555\",\n"
-            + "    \"idrDiagIcdTypeEnum\": \"DIAGNOSIS_ICD_TYPE_ICD9\",\n"
-            + "    \"idrDiagCode\": \"s2vgfx\"\n"
+            + "    \"idrDiagCode\": \"78\"\n"
             + "  }],\n"
             + "  \"mcsDetails\": [{\n"
-            + "    \"idrDtlFromDate\": \"2021-02-03\",\n"
-            + "    \"idrDtlToDate\": \"2021-04-24\",\n"
-            + "    \"idrProcCode\": \"6f\",\n"
-            + "    \"idrModOne\": \"zx\",\n"
-            + "    \"idrModTwo\": \"2\",\n"
-            + "    \"idrModThree\": \"6s\",\n"
-            + "    \"idrModFour\": \"g4\",\n"
-            + "    \"idrDtlDiagIcdTypeEnum\": \"DIAGNOSIS_ICD_TYPE_ICD9\",\n"
-            + "    \"idrDtlPrimaryDiagCode\": \"x9q\",\n"
-            + "    \"idrKPosLnameOrg\": \"hkjtsdcjmrtggtbqzrdqrmkbvfxhnvxzgbvnvjhrcsjprvjgnkkxdtrjgj\",\n"
-            + "    \"idrKPosFname\": \"bvcdd\",\n"
-            + "    \"idrKPosMname\": \"mr\",\n"
-            + "    \"idrKPosAddr1\": \"qhvqh3c96zdpt8521vrb9sdtkrmm3q85tsrxk06k06\",\n"
-            + "    \"idrKPosAddr21st\": \"rsz5gm0xt8m1fhkgs7130q\",\n"
-            + "    \"idrKPosAddr22nd\": \"5p\",\n"
-            + "    \"idrKPosCity\": \"ghqkzchskzft\",\n"
-            + "    \"idrKPosState\": \"nk\",\n"
-            + "    \"idrKPosZip\": \"77477992440000\",\n"
-            + "    \"idrDtlStatusUnrecognized\": \"b\"\n"
-            + "  }, {\n"
-            + "    \"idrDtlStatusEnum\": \"DETAIL_STATUS_SENT\",\n"
-            + "    \"idrDtlFromDate\": \"2021-01-19\",\n"
-            + "    \"idrDtlToDate\": \"2021-04-13\",\n"
-            + "    \"idrProcCode\": \"107gz\",\n"
-            + "    \"idrModOne\": \"n\",\n"
-            + "    \"idrModTwo\": \"x\",\n"
-            + "    \"idrModThree\": \"v\",\n"
-            + "    \"idrModFour\": \"vw\",\n"
+            + "    \"idrDtlStatusEnum\": \"DETAIL_STATUS_PENDING\",\n"
```

Comment:
Are these just randomly generated and copied into these tests? Seems a lot easier just to call the randomizer code, turn it into json programmatically, and pass that to the test. This _looks_ kinda manual?

Reply:
This is testing the random claim generator.  That expected JSON is taken from a run of the generator with a known seed value and compared to a run during the test to see if the output has changed from the known good one.

Reply:
Ah I see, thanks for explaining!